packages/engine/src/authoring/AuthoringState.tsx

```diff
@@ -499,13 +499,16 @@ export const getSourceSnapshot = (sourceID: SourceID) => {
       if (!sceneComponentID) continue
       // special case for entity tree since we need to serialize an EntityID
       if (component === EntityTreeComponent) {
+        const entityTree = getComponent(entity, EntityTreeComponent)
         // use an empty string to denote the root of the source
+        let parentEntityID = ''
+        if (entityTree.parentEntity !== sourceEntity && hasComponent(entityTree.parentEntity, UUIDComponent)) {
```

Comment:
all entities loaded into the authoring layer should have a UUIDComponent, we need to investigate why this is necessary if it really is